tests/unit/Endpoints/Webhooks/UpdateOrderTest.php

```diff
@@ -88,6 +89,17 @@ protected function executeOrderUpdateTest(): void
     {
         $this->syncShopInformation();
 
+        StoreKeeperApi::$mockAdapter->withModule(
```

Comment:
you should also assert if it's saved on order

Reply:
Added new assert and simple mock for `getOrder`.